test/ForceMove/_acceptableWhoSignedWhat.test.ts

```diff
@@ -1,8 +1,8 @@
 import {ethers} from 'ethers';
 import {expectRevert} from 'magmo-devtools';
 // @ts-ignore
-import ForceMoveArtifact from '../../build/contracts/TESTForceMove.json';
-import {setupContracts} from './test-helpers';
+import ForceMoveArtifact from '../../build/contracts/TESTNitroAdjudicator.json';
```

Comment:
I think it might be worth continuing to test `ForceMove` directly. It's entirely possible that it will be wrapped in other adjudicators at a later date.

Reply:
👍

Reply:
Addresses in 0fc73f6